src/index.ts

```diff
@@ -20,48 +24,57 @@ const run = async (): Promise<void> => {
       );
     }
 
-    let prData = await getData(authToken);
+    const coverageConfiguration: PathProperties[] = await getConfiguration();
```

Comment:
Does this work in the case where there is no config file?

Reply:
yeah for it defaults to the original way we got the data if there is no file found.